etc/picongpu/summit-ornl/gpu_batch.tpl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# Copyright 2019 Axel Huebl, Rene Widera
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for Summit's LSF (bsub) batch system
+#   https://www.olcf.ornl.gov/for-users/system-user-guides/summit/summit-user-guide/#running-jobs
+
+#BSUB -q !TBG_queue
+#BSUB -W !TBG_wallTimeNoSeconds
+# Sets batch job's name
+#BSUB -J !TBG_jobName
+#BSUB -nnodes !TBG_nodes
+#BSUB -alloc_flags smt4
+# send me mails on job (-B)egin, Fi(-N)ish
+#BSUB !TBG_mailSettings -u !TBG_mailAddress
+#BSUB -cwd !TBG_dstPath
+#BSUB -P !TBG_nameProject
+
+#BSUB -o stdout.%J
```

Comment:
are we requiring to name those `stdout`/`stderr` without a suffix these days? (post-processing scripts)

Reply:
no, we rely on `simOutput/output`